docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,847 @@
+{
```

Comment:
<p>"discovering POTENTIAL anomalies for the remaining points"</p><p> </p><p>"offline/batch setting"</p><p> </p><p>Change to <code>split_idx</code> (see below)</p><p> </p><p>"and the POTENTIAL anomalies should be coming from..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>